### PR TITLE
Add Yahoo Japan News case study

### LINKED ISSUE #4517: content: Yahoo! JAPAN [2021-01-29]
* [draft](https://docs.google.com/document/d/1ABHfxnv7Igg5fD61qxCu9UeNjIIBnm71NtGjWByJy30/edit?usp=sharing&resourcekey=0-4Yq4Qcj_1qlUdX5PR8mylg)
* [thread](https://groups.google.com/a/google.com/g/webdev-content/c/E_yR15SVF1c)

### PR DESCRIPTION
**Pull request by mihajlija**
Fixes #4517

cc @sisidovski

### CODE REVIEW

**Comment by sisidovski** [comment]
@mihajlija Thank you! I got hero images from Yahoo! JAPAN.

- image/iEYkFAI5LNQv7oLlIBv9DuB265a2/SYCpuPDC7QTj1runSA2C.png (376 x 240)
- image/iEYkFAI5LNQv7oLlIBv9DuB265a2/CT42V29q9DyjsktM03Wi.png (3200 x 960)

File: src/site/content/en/blog/yahoo-japan-news/index.md [hunk 1]
```diff
@@ -0,0 +1,216 @@
+---
+title: How CLS optimizations increased Yahoo! JAPAN News's page views per session by 15%
+subhead: How to get your PWA to show up next to platform-specific apps in system-level sharing UIs
```
Comment (kaycebasques) [inline src/site/content/en/blog/yahoo-japan-news/index.md:3]:
```suggestion
subhead: >
  Optimizing CLS by 0.2 led to a 15% increase in page
  views per session, 13% longer session durations, and a 1.72
  percentage point decrease in bounce rate.
```

@mihajlija make sure to catch copy-paste errors like this when you have admin privileges! ;P

File: src/site/content/en/blog/yahoo-japan-news/index.md [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+hero: image/iEYkFAI5LNQv7oLlIBv9DuB265a2/CT42V29q9DyjsktM03Wi.png
+thumbnail: image/iEYkFAI5LNQv7oLlIBv9DuB265a2/SYCpuPDC7QTj1runSA2C.png
+alt: 
+tags:
+  - blog
+  - case-study
+  - web-vitals
+---
+
+[Yahoo! JAPAN](https://www.yahoo.co.jp) is one of the largest media in Japan,
```
Comment (kaycebasques) [inline src/site/content/en/blog/yahoo-japan-news/index.md:23]:
```suggestion
[Yahoo! JAPAN](https://www.yahoo.co.jp) is one of the largest media companies in Japan,
```

File: src/site/content/en/blog/yahoo-japan-news/index.md [hunk 3]
```diff
@@ -0,0 +1,216 @@
...
+[aspect-ratio](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/aspect-ratio)
+property available in modern browsers. However, Yahoo! JAPAN News needed to
+implement images that also meet the following requirements:
+
+-  Prevent the aspect ratio from collapsing.
+-  Display images of various sizes (such as portraits and landscapes).
+-  Display images at 100% width of the parent element. However, images with
+   height over 300 px will be scaled down to 300 px height while preserving the
+   aspect ratio.
+-  Images can have links and be clickable.
```
Comment (kaycebasques) [inline src/site/content/en/blog/yahoo-japan-news/index.md:144]:
```suggestion
-  Make images clickable and support links.
```
@mihajlija check out the writing concept of parallelism: https://www.grammarly.com/blog/parallelism/

In this case all the other list items start with a verb so we can make this one more parallel by also starting with a verb

File: src/site/content/en/blog/yahoo-japan-news/index.md [hunk 4]
```diff
@@ -0,0 +1,216 @@
...
+-  Support not only modern browsers, but also browsers installed in
+   relatively old OS such as iOS 9.
```
Comment (kaycebasques) [inline src/site/content/en/blog/yahoo-japan-news/index.md:146]:
```suggestion
   relatively old operating systems such as iOS 9.
```

**Review by kaycebasques — APPROVED** [review]

File: src/site/content/en/blog/yahoo-japan-news/index.md [hunk 5]
```diff
@@ -0,0 +1,219 @@
...
+description: |
+    Using Search Console and Lighthouse to monitor Core Web Vitals, they discovered
+    opportunities for optimizing CLS score by 0.2 which led to 15% increase in page
+    views per session, 13% longer session duration, and 1.72 percentage point
+    decrease in bounce rate.
+authors:
+  - sisidovski
+  - gladenjoy
+  - mihajlija
+date: 2021-03-01
```
Comment (mihajlija) [inline src/site/content/en/blog/yahoo-japan-news/index.md:16]:
```suggestion
date: 2021-03-09
```

**Pull request merged**